func.html

```diff
@@ -85,64 +78,70 @@ <h1>Overview</h1>
 
-      pmr::memory_resource* get_memory_resource() const noexcept;
+      allocator_type get_allocator() const noexcept;
     };
 
   } // namespace experimental::inline fundamentals_v3
-
-  template &lt;class R, class... ArgTypes, class Alloc&gt;
-  struct uses_allocator&lt;experimental::function&lt;R(ArgTypes...)&gt;, Alloc&gt;
-    : true_type { };
-
 } // namespace std</code></pre>
 
     </cxx-section>
 
     <cxx-section id="func.wrap.func.con">
       <h1><code>function</code> construct/copy/destroy</h1>
 
-      <cxx-ednote>
-        The following first paragraph has been slightly editorially improved to
-        (a) make the difference between <code>std::function</code> and <code>std::experimental::function</code> clearer
-        (which seems necessary due to the "including" wording that can be parsed in two different ways) and
-        (b) to make intended normative wording clearer, that had been put into parenthesis before, by simply removing these parenthesis.
-      </cxx-ednote>
-
       <p>
-        When a <code>function</code> constructor that takes a first argument of type <code>allocator_arg_t</code> is invoked,
-        the second argument is treated as a <cxx-term>type-erased allocator</cxx-term> (<cxx-ref to="memory.type.erased.allocator"></cxx-ref>).
-        If the constructor moves or makes a copy
-        of a function object (<cxx-ref in="cxx" to="function.objects"></cxx-ref>),
-        including an instance of the <code>experimental::function</code> class template,
-        then that move or copy is performed by <cxx-term>using-allocator construction</cxx-term> with allocator <code>get_memory_resource()</code>.
+        A function object stores an allocator object of type <code>std::pmr::polymorphic_allocator&lt;&gt;</code>,
+        which it uses to allocate memory for its internal data structures.
+        In the <code>function</code> constructors, the allocator is initialized
+        (before the target object, if any) as follows:
       </p>
-
+      <ul>
+        <li>
+          For the move constructor, the allocator is initialized
+          from <code>f.get_allocator()</code>, where <code>f</code> is the
+          parameter of the constructor.
+        </li>
+        <li>
+          For constructors having a first parameter of type <code>allocator_arg_t</code>,
+          the allocator is initialized from the second parameter.
+        </li>
+        <li>
+          For all other constructors, the allocator is value-initialized.
+        </li>
+      </ul>
       <p>
-        In the following descriptions, let <code><em>ALLOCATOR_OF</em>(f)</code> be the allocator specified in the construction of <code>function</code> <code>f</code>,
-        or the value of <code>experimental::<wbr>pmr::<wbr>get_default_resource()</code> at the time of the construction of <code>f</code> if no allocator was specified.
+        In all cases, the allocator of a parameter having type <code>function&&</code> is unchanged.
```

Comment:
```suggestion
        In all cases, the allocator of a parameter having type <code>function&amp;&amp;</code> is unchanged.
```

Is this needed?

Reply:
No, it doesn't actually seem necessary, but it's still polite, so I'll accept it!